flood/spark/transforms.py

```diff
@@ -0,0 +1,191 @@
+from pyspark.sql.types import DoubleType
+from pyspark.sql import functions as F
+from pyspark.sql.window import Window
+
+def round_to_precision(value, precision):
+    """
+    Round a value to the given number of decimal places.
+    """
+    return round(value, precision)
+
+def create_round_udf(precision=3):
+    """
+    Create a UDF for rounding to the specified precision.
+    """
+    return F.udf(lambda x: round_to_precision(x, precision), DoubleType())
```

Comment:
I would remove this and replace the usage with `F.round`.

Reply:
Done